data/mods/Magiclysm/monsters/lizardfolk.json

```diff
@@ -0,0 +1,123 @@
+    "morale": 80,
+    "vision_day": 10,
+    "vision_night": 20,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 0,
+    "dodge": 5,
+    "armor_bash": 2,
+    "armor_cut": 12,
+    "armor_stab": 6,
+    "harvest": "lizardfolk",
+    "death_function": [ "NORMAL" ],
+    "death_drops": [ 
+	  { "item": "lizardfolk_club", "prob": 30 },
+	  { "item": "loincloth", "prob": 40 },
+	  { "item": "leather_belt", "prob": 20 },
+	  { "item": "bone_human", "prob": 10 }
+	],
+    "special_attacks": [ [ "crush", 5 ], { "type": "bite", "cooldown": 5 } ],
+    "flags": [
+      "SEES", "HEARS", "SMELLS", "KEENNOSE", "PATH_AVOID_DANGER_1", "CLIMBS", "GROUP_MORALE", "LEATHER",
+      "SWIMS", "ATTACKMON", "PUSH_MON", "SWARMS", "BONES", "CAN_OPEN_DOORS", "NO_BREATHE"
+    ]
+  },
+  { 
+	"id": "mon_lizardfolk_hunter",
+    "type": "MONSTER",
+    "name": "lizardfolk hunter",
+    "description": "Lizardfolk are very intelligent and cunning, but magical ability is a rare quality.  Shamans are chosen from the tribe in childhood, when magical abilities mark the fate of the young tribesman.  Not much is known about the initiation ritual they must undergo, but few survive the experience.  Shamans are druidic spellcasters that can use the forces of nature to battle enemies as well as summoning assistance when needed.",
+    "copy-from": "mon_lizardfolk_warrior",
+    "symbol": "H",
+    "special_attacks": [
+	  {
+        "type": "gun",
+        "cooldown": 5,
+        "move_cost": 150,
+        "gun_type": "lizardfolk_barbed_dart",
+        "fake_skills": [ [ "gun", 8 ], [ "rifle", 8 ] ],
+        "fake_dex": 8,
+        "fake_per": 8,
+        "require_targeting_player": false,
+        "ranges": [ [ 2, 10, "DEFAULT" ] ],
+        "description": "The hunter hurls a barbed dart at you!"
+	  },		  
+	  { "type": "bite", "cooldown": 5 }
+	],
+    "death_drops": [ 
+	  { "item": "lizardfolk_javelin", "prob": 30 },
+	  { "item": "loincloth", "prob": 40 },
+	  { "item": "spearsling", "prob": 20 },
+	  { "item": "bone_human", "prob": 10 }
+	]
+  },
+  { 
+	"id": "mon_lizardfolk_shaman",
+    "type": "MONSTER",
+    "name": "lizardfolk shaman",
+    "description": "Lizardfolk are very intelligent and cunning, but magical ability is a rare quality.  Shamans are chosen from the tribe in childhood, when magical abilities mark the fate of the young tribesman.  Not much is known about the initiation ritual they must undergo, but few survive the experience.  Shamans are druidic spellcasters that can use the forces of nature to battle enemies as well as summoning assistance when needed.",
+    "copy-from": "mon_lizardfolk_warrior",
+    "symbol": "S",
+	"hp": 80,
+    "melee_skill": 3,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "special_attacks": [ { "type": "spell", "spell_id": "spell_shaman_base", "cooldown": 5 }, [ "scratch", 5 ] ],
+    "death_drops": [ 
+	  { "group": "demon_spider_lair_treasure", "prob": 30 },
```

Comment:
no, this should definitely be its own item group or some other item grou. demon spider lair treasure being dropped by a lizardfolk doesn't make sense.